src/injected/analyzer-controller.ts

```diff
@@ -72,8 +72,8 @@ export class AnalyzerController {
     private onResultsChangedState = (): void => {
         const state = this.visualizationResultsStore.getState();
         if (state.tabStops.tabbingCompleted && state.tabStops.needToCollectTabbingResults) {
-            this.tabStopRequirementActionMessageCreator.updateNeedToCollectTabbingResults(false);
             this.teardown(AdHocTestkeys.TabStops);
+            this.tabStopRequirementActionMessageCreator.updateNeedToCollectTabbingResults(false);
```

Comment:
The change itself looks good, but can we add test coverage to make sure this doesn't regress again?